lib/modules/manager/npm/post-update/pnpm.ts

```diff
@@ -153,3 +153,63 @@ async function getPnpmConstraint(
   }
   return result;
 }
+
+export function getConstraintsFromLockFile(lockfileVersion: number): string {
```

Comment:
where does it get called? 🤔

shouldn't it be called in function above where also lockfile is handled?